Doc fix qceff-examples & netcdf inflation files

hkershaw-brown opened the pull request:
## Description:
<!--- Describe your changes -->
Removes out-of-date reference to quantile-methods branch
Removes references to non-existant netcdf_inflation_files.rst (removed in #698) 

### Fixes issue
<!--- link to github issue(s) -->
fixes #710 

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

### Documentation changes needed?
<!-- Put an `x` in all the boxes that apply: -->
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.

### Tests
Documentation build

## Checklist for merging

- [ ] Updated changelog entry
- [ ] Documentation updated
- [ ] Update conf.py

## Checklist for release
- [ ] Merge into main
- [ ] Create release from the main branch with appropriate tag
- [ ] Delete feature-branch

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [ ] No dataset needed

kdraeder commented:
Sorry that I didn't respond to this earlier.  (I was busy reading these docs and adapting reanalysis branch scripts to use qceff capabilities).

Removing refs to the quantile_methods is good.

I also wondered whether the qceff table replaces the assim_tools_nml method described in the "previous" page,
in which case that file should be updated, maybe as part of this PR.

A link to qceff_examples in qceff_probit would have been useful to me,
although I found it through a search.

In the qceff_probit.rst and the qceff-examples.rst I recommend changing "anxious" to "eager"

hkershaw-brown commented:
more and more duplicate docs:

https://dart-documentation--711.org.readthedocs.build/en/711/guide/filters.html#filters

hkershaw-brown commented:
and more: https://docs.dart.ucar.edu/en/latest/guide/important-capabilities-dart.html#data-assimilation-algorithms-available-in-dart

kdraeder commented on guide/important-capabilities-dart.rst:
```diff
@@ -144,10 +144,9 @@ Data assimilation algorithms available in DART
 algorithms for filtering, inflation/deflation, and covariance localization.
 
-DART offers numerous **filter algorithms**. These determine how the posterior
+DART offers numerous **filter algorithms** in the :ref:`QCEFF`. These determine how the posterior
```
I believe "posterior" should be "prior".  The prior is updated by the algorithm to yield the posterior.   And then remove "and the prior ensemble"

hkershaw-brown replied:
Thanks for checking this Kevin. changed to your suggestion.  Better?

kdraeder replied:
I see the fix in "Commits", but not in "Files Changed".
The fix looks good.

kdraeder commented:
> more and more duplicate docs:
> 
> https://dart-documentation--711.org.readthedocs.build/en/711/guide/filters.html#filters

I was going to approve the PR, but then remembered these 2 notes about duplicates.
Are there fixes on the way that I should review?

hkershaw-brown commented:
> > more and more duplicate docs:
> > https://dart-documentation--711.org.readthedocs.build/en/711/guide/filters.html#filters
> 
> I was going to approve the PR, but then remembered these 2 notes about duplicates. Are there fixes on the way that I should review?

yeah let me push em up.

hkershaw-brown commented:
@kdraeder 
  removed filter.rst file (seems pointless) and removed that file from table of contents
  removed filter_kind  assim_tools_nml comment from guide/important-capabilities-dart.rst
  removed filter_kind from example assim_tools_nmls in:
     - models/clm/tutorial/README.rst
     - models/cm1/readme.rst

That was all the filter_kind assim_tools_nml references I could find.

kdraeder commented on models/clm/tutorial/README.rst:
typo; witin -> within

kdraeder replied:
This is not a show-stopper, and the other recent changes look fine.

hkershaw-brown replied:
I fixed it, then put a typo in my commit message. Conservation of typos.

kdraeder reviewed (approved):
Onward and sideways!  There's probably a way to edit comments, but I wouldn't bother.
This issue is wrapped up well enough for me!
